packages/es-components/src/index.js

```diff
@@ -17,6 +17,7 @@ export DropdownButton from './components/controls/buttons/DropdownButton';
 export RadioGroup from './components/controls/radio-buttons/RadioGroup';
 export Dropdown from './components/controls/dropdown/Dropdown';
 export Checkbox from './components/controls/checkbox/Checkbox';
+export CheckboxLabel from './components/controls/checkbox/CheckboxLabel';
```

Comment:
Does `CheckboxLabel` belong in `controls`? Since the user isn't really doing anything with the label it seems like `containers` would be more appropriate.